packages/harpguru-core/src/components/scales-menu/scales-menu.tsx

```diff
@@ -31,7 +52,11 @@ export const ScalesMenu = (menuProps: MenuProps): React.ReactElement => {
     [dispatchAndFlushScaleToggles]
   )
 
-  const optionStackProps = getOptionStackProps(useSubTitle, itemTapHandler)
+  const optionStackProps = getOptionStackProps(
```

Comment:
Probably this should be rearranged to resemble whatever we come up with for the covariant and layout menus.

Reply:
Done